CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Version 4.4.3 (Under active development)
 
+### App Center Distribute
+
+* **[Fix]** Fix checking a new release if the application was already updated before.
```

Comment:
"Fix installing a new release"?

Reply:
Not sure that this is correct. Primarily, SDK can't check that the new release exists. 
> Not checking in app updates as installed from a store.

That the SDK can't install it is already the consequence.